src/modules/blocks/rsvp/inactive-block/template.js

```diff
@@ -19,35 +24,31 @@ const RSVPInactiveBlock = ({ created }) => {
 		: __( 'Allow users to confirm their attendance.', 'event-tickets' );
 
 	return (
-		<div class="tribe-common tribe-editor__inactive-block--rsvp">
-			<div class="tribe-editor__inactive-block--rsvp__container">
-				<div class="tribe-editor__rsvp tribe-common-g-row tribe-common-g-row--gutters">
-					<div class="tribe-editor__rsvp-details-wrapper tribe-common-g-col">
-						<div class="tribe-editor__rsvp-details">
-							<h3 class="tribe-editor__rsvp-title tribe-common-h2 tribe-common-h4--min-medium">
-								{title}
-							</h3>
-
-							<div class="tribe-editor__rsvp-description tribe-common-h6 tribe-common-h--alt tribe-common-b3--min-medium">
-								{description}
-							</div>
-						</div>
+		<Card className="tribe-common tribe-editor__inactive-block--rsvp">
+			<div className="tribe-editor__rsvp-details-wrapper">
+				<div className="tribe-editor__rsvp-details">
+					<h3 className="tribe-editor__rsvp-title tribe-common-h2 tribe-common-h4--min-medium">
+						{title}
+					</h3>
+
+					<div className="tribe-editor__rsvp-description tribe-common-h6 tribe-common-h--alt tribe-common-b3--min-medium">
+						{description}
 					</div>
+				</div>
+			</div>
 
-					<div class="tribe-editor__rsvp-actions-wrapper tribe-common-g-col">
-						<div class="tribe-editor__rsvp-actions">
-							<div class="tribe-editor__rsvp-actions-rsvp">
-								<div class="tribe-editor__rsvp-actions-rsvp-create">
-									<button class="tribe-common-c-btn tribe-common-b1 tribe-common-b2--min-medium">
-										{ __( 'Create RSVP', 'event-tickets' )}
-									</button>
-								</div>
-							</div>
+			<div className="tribe-editor__rsvp-actions-wrapper">
+				<div className="tribe-editor__rsvp-actions">
+					<div className="tribe-editor__rsvp-actions-rsvp">
+						<div className="tribe-editor__rsvp-actions-rsvp-create">
+							<button className="tribe-common-c-btn tribe-common-b1 tribe-common-b2--min-medium">
+								{ __( 'Add RSVP', 'event-tickets' )}
```

Comment:
For uniformity's sake, should we store this text value as a variable and use that instead? Not necessary, but just a thought.

Reply:
@codingmusician By using `__` the text can be translatable with a Pot file, tickets block were already using this pattern for text, we just need to have those Pot files in another jira ticket probably